toolkit/offchain/src/csl.rs

```diff
@@ -146,6 +146,10 @@ pub(crate) trait OgmiosUtxoExt {
 	fn to_csl_tx_input(&self) -> TransactionInput;
 	fn to_csl_tx_output(&self) -> Result<TransactionOutput, JsError>;
 	fn to_csl(&self) -> Result<TransactionUnspentOutput, JsError>;
+
+	/// Encodes this UTXO as a nested constructor data format which is used by PC smart contracts
+	fn to_constructor_style_plutus_data(&self) -> uplc::PlutusData;
+	fn to_constructor_style_csl_plutus_data(&self) -> PlutusData;
```

Comment:
We have just `to_datum` for our domain type. It is not a style, it is a way we represent UtxoId as PlutusData. I suggest renaming it to `to_uplc_plutus_data` and `to_csl_plutus_data`.